lib/event.js

```diff
@@ -8,4 +8,18 @@ module.exports = class SDLEvent {
   get type() {
     return binding.getEventType(this._handle)
   }
+
+  get key() {
+    return new SDLEventKey(this._handle)
```

Comment:
I'm not sure if returning a class instance here is the best option. I've also not yet thought of other options :)